build.gradle.kts

```diff
@@ -5,10 +5,10 @@ buildscript {
         mavenCentral()
     }
     dependencies {
-        classpath("org.jetbrains.kotlin:kotlin-gradle-plugin:1.5.0")
-        classpath("com.android.tools.build:gradle:7.2.0")
-        classpath("org.jetbrains.dokka:dokka-gradle-plugin:1.5.0")
-        classpath("org.jetbrains.dokka:dokka-core:1.5.0")
+        classpath("org.jetbrains.kotlin:kotlin-gradle-plugin:1.9.0")
+        classpath("com.android.tools.build:gradle:7.4.1")
+//        classpath("org.jetbrains.dokka:dokka-gradle-plugin:1.8.20")
```

Comment:
Can these be removed?

Reply:
this is for Dokka we will use it back once we got any response from them about Kotlin `1.9.0` support